packages/schema/src/remove.rs

```diff
@@ -0,0 +1,35 @@
+use std::{fs, io, path};
+
+fn is_regular_file(path: &path::Path) -> Result<bool, io::Error> {
```

Comment:
Why not return a `bool` here like you do in the other checks (`.unwrap_or(false)`)?

It is fine either way but nice to be consistent.

Reply:
For errors, the function `is_regular_file` cannot give and answer and the caller should decide what do to with it.

In the other cases we know that when `.file_name()` is `None`, the path is neither hidden nor a .json. There we just silently ignore conversion errors from OsString to str, which I think should panic.

Reply:
Even better, with https://doc.rust-lang.org/std/ffi/struct.OsStr.html#method.to_string_lossy we don't have any error case in the other two functions

Reply:
Okay, fair enough. Feel free to merge

Reply:
Added two commits to make is_hidden and is_json better